lib/make-sass-renderer.js

```diff
@@ -27,12 +29,22 @@ module.exports = function(inputFileName, assetManager, sourcemaps, compact) {
 // promisified version of sass.render
 function renderSass(options) {
 	return new Promise((resolve, reject) => {
-		sass.render(options, (err, result) => {
-			if(err) {
-				reject(err);
-			} else {
-				resolve(result);
-			}
-		});
+		try {
+			// using synchronous rendering, because it is faster
+			let result = sass.renderSync(options);
+			result.css = fixEof(result.css);
+			resolve(result);
+		} catch(e) {
+			reject(e);
+		}
 	});
 }
+
+// In Unix, every file shall end with a newline. sass doesn't seem to care.
+function fixEof(buf) {
```

Comment:
* "On Unix"? I'd just drop that bit entirely though
* "should have a trailing blank line"?
* `s/sass/Sass/`
* `s/Eof/EOF/`

Reply:
`sass` is correct in this context, because it is the name of the package. Not the language.

Reply:
Ah, makes sense - that's not obvious though... Oh well, it's not that important.